mne/preprocessing/ica.py

```diff
@@ -906,28 +908,19 @@ def _sources_as_raw(self, raw, add_channels, start, stop):
             raw._data = data
 
         # populate copied raw.
-        start, stop = _check_start_stop(raw, start, stop)
         if add_channels is not None:
             raw_picked = raw.copy().pick_channels(add_channels)
-            data_, times_ = raw_picked[:, start:stop]
+            data_, _ = raw_picked[:, start:stop]
```

Comment:
did you consider using raw_picked._getitem(..., return_times=False) here?

Reply:
Nope, added that after this change -- will push a commit